packages/paste-website/src/pages/getting-started/design/index.mdx

```diff
@@ -0,0 +1,400 @@
+import {Tooltip} from '@twilio-paste/tooltip';
+import {Flex} from '@twilio-paste/flex';
+import {version as typesVersion} from '@twilio-paste/types/package.json';
+import {version as stylePropsVersion} from '@twilio-paste/style-props/package.json';
+import {version as designTokensVersion} from '@twilio-paste/design-tokens/package.json';
+import {version as themeVersion} from '@twilio-paste/theme/package.json';
+import {version as coreVersion, dependencies as coreDependencies} from '@twilio-paste/core/package.json';
+import {version as iconsVersion} from '@twilio-paste/icons/package.json';
+import {version as animationVersion} from '@twilio-paste/animation-library/package.json';
+import {version as stylingLibraryPropsVersion} from '@twilio-paste/styling-library/package.json';
+import {version as dropdownVersion} from '@twilio-paste/dropdown-library/package.json';
+import {version as reakitVersion} from '@twilio-paste/reakit-library/package.json';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {InlineCode} from '../../../components/Typography';
+import Package from '../../../../../../package.json';
+
+import { SuccessIcon } from "@twilio-paste/icons/esm/SuccessIcon";
+import { WarningIcon } from "@twilio-paste/icons/esm/WarningIcon";
+import { ErrorIcon } from "@twilio-paste/icons/esm/ErrorIcon";
+
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/getting-started/design/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Installation
+
+You’ll need to download a few things before you can get started using Paste in
+your design work.
+
+### Abstract
+
+All product designers across Twilio use <Anchor href="https://www.abstract.com/">Abstract</Anchor> to host and version control
+their design files. We also use Abstract to link Paste Sketch libraries and
+manage library updates (more on that later). Connect with your design manager
+to get added to the Twilio team in Abstract.
+
+<Callout variant="information">
+  <CalloutTitle>
+    Getting familiar with Abstract
+  </CalloutTitle>
+  <CalloutText>
+    If you are not yet familiar with Abstract, please take some time to learn
+    about <Anchor href="https://www.abstract.com/how-it-works">how it works</Anchor>,
+    and read through these resources
+    on <Anchor href="https://help.abstract.com/best-practices/">best practices</Anchor>.
+  </CalloutText>
+</Callout>
+
+### Sketch
+
+Product designers across Twilio use Sketch to design their UIs. You
+can <Anchor href="https://twilio.service-now.com/sp/">file a ticket with IT</Anchor> to
+get a Sketch license. In the meantime, you
```

Comment:
Remove the free license mention. It reads pretty badly from an external company perspective.